next/components/search/search-multicheckbox-facet.tsx

```diff
@@ -21,10 +21,10 @@ export function MultiCheckboxFacet({
 }: FacetViewProps) {
   const { t } = useTranslation();
   return (
-    <fieldset className={clsx(className, "mb-4")}>
-      <legend className="mb-5 text-heading-xs font-bold text-steelgray">
+    <section className={clsx(className, "mb-4")}>
+      <div className="mb-5 text-heading-xs font-bold text-steelgray">
         {label}
-      </legend>
+      </div>
       {showSearch && (
```

Comment:
I think we can also remove this showSearch prop and what it renders - it's not used at all afaik. (The search box is always visible at the top of the page, we'll never render it here with the filters)